### DISCUSSION TITLE
增加屏蔽”值几分/打分/评星/星级/评价“弹窗的功能

### DISCUSSION THREAD

**Discussion by wadecoding** [the1812/Bilibili-Evolved#2067] [功能建议 / Ideas]
播放器版本`2.62.3-92d0d613`，在[这个视频](https://www.bilibili.com/video/BV17g411M7qV)的05:58附近出现了评星的弹窗，选择器为`.bilibili-player-score.bilibili-player-score-show.bilibili-player-score-show-animate`。能不能在下个版本中增加屏蔽该弹窗的功能，先提前谢谢了。

**Comment by Coulomb-G** [comment]
要是急的话,推荐使用stylus之类,版本一般个把月更新次

**Reply by wadecoding** [reply]
不急，如果项目不复杂我都想提个PR自己加feature了，但看完贡献指南后放弃了…

**Reply by the1812** [reply]
可以先手动加一下, 浏览器控制台里
```js
bilibiliEvolved.installStyle({
  name: 'hideVideoRate',
  displayName: '隐藏视频评分',
  style: `.bilibili-player-score { display: none !important }`,
})
```

**Comment by Geeyun-JY3** [comment]
重复了哟：https://github.com/the1812/Bilibili-Evolved/discussions/1332#discussioncomment-991709
不过另开也没什么问题

**Reply by the1812** [reply]
带了很准确的信息, 另开一个也很河里(

**Reply by Geeyun-JY3** [reply]
也是，#1332 这标题也不太适合统一反馈

**Reply by wadecoding** [reply]
我提这个之前搜了一下，浏览了标题觉得和我遇到的不一样所以就另开了一个…
